examples/local_elasticsearch.conf

```diff
@@ -0,0 +1,26 @@
+input {
+  http_poller {
+    urls => {
```

Comment:
historically I have tried to avoid complex configuration settings like this. Worries I have - Is the complexity worth it? Can we document it well enough?